src/main/java/be/seeseemelk/mockbukkit/scheduler/BukkitSchedulerMock.java

```diff
@@ -88,6 +91,11 @@ else if (!task.isCancelled())
 				tasks.add(task);
 			}
 		}
+		if (oldTasks.size() == 0 && asyncTasksQueued > 0)
+		{
+			Logger.getLogger(LOGGER_NAME).warning("Asynchronous Tasks may have been lost -> " + asyncTasksQueued);
```

Comment:
I'm not a big fan of this warning message. I can imagine that if you write a unit test, and see this in your logs, many would either never see it or be confused why this cryptic error message appears.

Reply:
Basically - the fact was that this is an implementation issue with the currently sheduler it should never happen in a real server.    There is no separate queue and task count they are the same.    So I can remove the log message.